fc/consent.go

```diff
@@ -0,0 +1,72 @@
+package fullcontact
+
+type ConsentPurposesOption func(consentPurposes *ConsentPurposes)
+
+type ConsentPurposes struct {
```

Comment:
Can you include this code in permission_request.go, as now we have both consent.go and permission.go which can be confusing. As this is a request, we can include this in permission_request.go

Reply:
Done. Moved the code to permission_request.go and also renamed it to ConsentPurpose